creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -137,7 +155,15 @@ const Metrics = Object.freeze({
         }
       }
     ])
-  )
+  ),
+  [MetricNames.FIND_SYNCS_RESULTS_TOTAL_GAUGE]: {
```

Comment:
can you give better name like `FIND_SYNC_REQUESTS_JOB_RESULTS_COUNTS_GAUGE` or `FIND_SYNC_REQUEST_COUNTS_GAUGE` (note request is singular vs plural)

Reply:
changed to `FIND_SYNC_REQUEST_COUNTS_GAUGE` singular